docker/pandoc-lambda/function/app.py

```diff
@@ -52,7 +171,19 @@ def handler(event, context):
                         sig_string = f"unknown signal {-e.returncode}"
                 else:
                     sig_string = f"non-zero exit status {e.returncode}"
-                raise Exception(f"Pandoc command exited with {sig_string}")
+                ss = "Pandoc command exited with " + str(sig_string)
+                print(ss)
+                pandoc_out.seek(0,0)
+                print(response.stderr)
+                raise Exception(ss)
```

Comment:
Do you still want the prints and seek here? Maybe debugging stuff that got left in by accident?

Do you happen to remember if `print(response.stderr)` ever came in handy here? I would have expected the process to exit before reaching this because of `if response.stderr: raise Exception(`, but I've found this error handling to be somewhat finicky......

Reply:
Yeah, these lines were debugging artifacts and I'll remove them. The print did not produce anything.